command/agent.go

```diff
@@ -107,6 +107,10 @@ Nomad Options:
 
 Policy Options:
 
+  -policy-dir=<path>
+    The path to a directory used to load scaling policies. Currently only cluster
+    scaling policies can be loaded by this method.
```

Comment:
Now that I am thinking about it, this could be a problem where the same task group is being target in the jobspec and in a file. For now let's pretend it doesn't happen 😬